workspaces/tekton/plugins/tekton/src/components/PipelineRunLogs/PipelineRunLogViewer.tsx

```diff
@@ -74,3 +72,24 @@ export const PipelineRunLogViewer = ({ pod }: PipelineRunLogViewerProps) => {
     </>
   );
 };
+
+const queryClient = new QueryClient({
```

Comment:
Wouldn't there be a benefit of using this as default ?

Reply:
The default client for handling fetches.

Reply:
Discussed offline, we'll save this for when we do a larger refactor.